server/sockets/list.js

```diff
@@ -477,6 +477,102 @@ const changeListType = (socket, dashboardClients, cohortClients, listClients) =>
       });
   });
 
+const emitListsOnRemoveCohortMember = (socket, dashboardClients, listClients) =>
+  socket.on(CohortActionTypes.REMOVE_MEMBER_SUCCESS, data => {
+    const { cohortId, userId } = data;
+
+    List.find({ cohortId })
+      .populate('cohortId', 'memberIds')
+      .lean()
+      .exec()
+      .then(docs => {
+        if (docs) {
+          const {
+            cohortId: { memberIds: cohortMemberIds }
+          } = docs[0];
+          const listIdsUserWasRemovedFrom = [];
+          const listIdsUserRemained = [];
+
+          docs.forEach(doc => {
+            const { type, viewersIds } = doc;
+            const isListViewer = checkIfArrayContainsUserId(viewersIds, userId);
```

Comment:
Consider separating isListViewer implementation and importing just that. It would be nice to have only `const listViewer = isListViewer(viewersIds)(userId)` - the internal implementation doesn't matter.